packages/context/test/unit/inject.test.ts

```diff
@@ -22,7 +22,8 @@ describe('function argument injection', () => {
     }
 
     const meta = describeInjectedArguments(TestClass);
-    expect(meta).to.deepEqual(['foo']);
+    expect(meta.length).to.eql(1);
+    expect(meta[0].bindingKey).to.deepEqual('foo');
```

Comment:
```ts
expect(meta.map(m => m.bindingKey)).to.deepEqual(['foo']);
```

This way the assertion error message can include extra keys when the meta has more than one entry.